### PR TITLE
Fix runtime in floor_damage.dm

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

image

image

## Why It's Good For The Game

Fixes cool.

## Changelog
:cl:

fix: runtime in floor_damage.dm

/:cl:

### CODE REVIEW

File: code/game/turfs/simulated/floor_damage.dm [hunk 1]
```diff
@@ -27,53 +27,42 @@
...
 
 	//The tile has broken!
-	if (health <= 0)
+	if(health <= 0)
 		//Leftover damage will carry over to whatever tile replaces this one
 		var/leftover = abs(health)
 		make_plating() //Destroy us and make the plating underneath
 		spawn()
-			//We'll spawn off a new stack in order to damage the next layer, incase it turns into a different turf object
-			damage_floor_at(x,y,z,leftover, damage_type, ignore_resistance)
```
Comment (hyperioo) [inline code/game/turfs/simulated/floor_damage.dm:56]: Indent after the spawn is gone

Comment (SirRichardFrancis) [reply code/game/turfs/simulated/floor_damage.dm:56]: Yep.

Comment (TheShown911) [reply code/game/turfs/simulated/floor_damage.dm:56]: Why have the spawn then?

Comment (TheShown911) [reply code/game/turfs/simulated/floor_damage.dm:56]: Oh it's still indented further down.

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**